tests/query_interface_test.js

```diff
@@ -440,7 +440,8 @@ describe('QueryInterface', () => {
     });
   });
 
-  describe('constraints', () => {
+  // Reason: gives this error on the before each: column "uid" does not exist. Works fine on CI
```

Comment:
Do you know why does this passes on CI but not locally?

Reply:
Removed unrelated skip.